packages/blade/src/utils/getMediaQuery/getMediaQuery.test.ts

```diff
@@ -0,0 +1,13 @@
+import { getMediaQuery } from './getMediaQuery';
+
+describe('getMediaQuery', () => {
+  it(`should return only min-width when only min is passed`, () => {
+    expect(getMediaQuery({ min: 1024 })).toBe('screen and (min-width: 1024px)');
```

Comment:
shouldn't this accept breakpoint tokens instead of tokens?

Reply:
It is low-level utilty so should be fine right? keeping it values keeps the logic more simpler in `useBreakpointValues` hook because we want to do minus of last value there.